task/03-date-tasks.js

```diff
@@ -56,7 +58,14 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
```

Comment:
please fix block (i mean if and else if blocks) formatting according to google code style https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks